README.md

```diff
@@ -98,6 +99,45 @@ Xcode is also able to display `TODO` and `FIXME` tags directly in the source nav
 
 Other conventional comment tags, such as `NOTE` are not recognized by Xcode.
 
+### Types ###
+
+Always use Swift native types when available. Swift offers a bridging to Objective-C so you can use them whenever you want.
+
+**Preferred:**
+
+```
+class myClass {
+...
+}
+```
+
+**Not preferred:**
+
+```
+@objc class myClass {
+...
+}
+```
+
+For numbers NSNumber is even less useful than before since you can create a Swift raw number type from any other types.
```

Comment:
I think we can be more generic about this instead of being very explicit about NSNumber:

`If you need functionality from an Objective-C type that is not available in its corresponding Swift type (for instance, needing an NSString function that is not available on String), cast your Swift raw type to the corresponding type instead of declaring the type as the Objective-C type. Be sure to cast back to the Swift type when this operation is completed.`